src/core/currency/wallet/currency-wallet-export.js

```diff
@@ -75,7 +75,7 @@ export function searchStringFilter(
       )
         return true
     }
-    if (tx.swapData != null && tx.swapData.plugin != null) {
+    if (tx.swapData != null) {
```

Comment:
Did you mean `tx.swapData?.plugin != null`?

Reply:
That was the original replacement, yes, but Flow blew up here for some reason (it works fine in the other places). I poked around in the code, and since `swapData` comes from a cleaner, `plugin` can never be null anyhow.

Reply:
I see. That makes sense.